flink-connector/flink/src/test/java/org/apache/gravitino/flink/connector/integration/test/FlinkCommonIT.java

```diff
@@ -280,6 +281,69 @@ public void testCreateSimpleTable() {
         supportDropCascade());
   }
 
+  @Test
+  @EnabledIf("supportTableOperation")
+  public void testCreateTableWithPrimaryKey() {
```

Comment:
I'm afraid Hive or Iceberg doesn't support Specifying Primary key.  maybe we could move this test to Paimon IT only.